sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -388,6 +411,49 @@ make_buffer(
 Construct a SYCL buffer instance from a pointer to a Level Zero memory allocation. Please refer to <code>make_buffer</code>
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
+</tr>
+
+<tr>
+<td>
+
+``` C++
+make_image(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          image<Dimensions, AllocatorT>> &,
+    const context &Context, event AvailableEvent = {})
+```
+</td>
+<td>This API is available starting with revision 4 of this specification.
+
+Construct a SYCL image instance from a ze_image_handle_t. 
+
+Because LevelZero has no way of getting image information from an image, it must be provided.  The  <code>backend_input_t</code> is a struct type like so:
+``` C++
+struct type {
+    ze_image_handle_t ZeImageHandle;
+    sycl::image_channel_order ChanOrder;
+    sycl::image_channel_type ChanType;
+    sycl::range<Dimensions> Range;
+    ext::oneapi::level_zero::ownership Ownership{
+        ext::oneapi::level_zero::ownership::transfer};
+  };
+```
+where the Range should be ordered (width), (width, height), or (width, height, depth) for 1D, 2D and 3D images respectively, 

```

Comment:
This order seems backwards w.r.t. to other uses of multi-dimensional ranges.  In SYCL , the rightmost dimension varies fastest (see [section 3.11.1](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:multi-dim-linearization)).  Wouldn't this correspond to "width"?

Reply:
That is not the order images use in SYCL as they were implemented in SYCL 1.2.1  In years past there was a lot of discussion about this, because it was weird for the ranges to be interpreted one way for buffers but another for images. That's just the way it ended up. IIRC, the computcpp implementation did the same thing. Haven't checked it recently.

I will add a note that this interpretation of range may be interim.

Since we are talking about it, I find the terminology "dimension that varies fastest" to be confusing. Wouldn't that be interpreted as the z dimension, not the x?  Because a +1 change in the z would cause the biggest jump in the linear position.  Talking about ranges abstractly is always challenging in the context of buffers and images, where they are no longer abstract.  I prefer the terminology of "contiguous memory". Enumerating the x term results in visiting contiguous memory. And, even better, I think we shouldn't shy away from using depth/z, height/y, width/x when talking about buffers and images.  That's the metaphor users are using in their head, the docs should follow suit.   Besides, our ranges aren't truly N-space anyway, are they? There is no `range<4>`  .

Reply:
Yes, I agree.  It's more important that the `range` order matches the order we have already in the `image` constructor.  If the `image` constructor uses the the first dimension for "width" we should do the same here.